Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -268,12 +268,22 @@
   categories:
     - UplinkUtility
 
-#TODO: Increase the price of this to 4-5/remove it when we get encrpytion keys
+#TODO: Increase the price to 4 when flashbang prof.
 - type: listing
   id: UplinkHeadset
   name: uplink-headset-name
   description: uplink-headset-desc
   productEntity: ClothingHeadsetAltSyndicate
+  cost:
+    Telecrystal: 3
+  categories:
+  - UplinkUtility
```

Comment:
Headset have 4 slots for keys, but since many keys have many channels and no need to use others like "Common key"...
We have 2 ways.
1. Remake keys and make them have some channels and put in headsets' many keys like "Command channel key", "Engineering channel key", so they will have more key slots...
2. I forgor 💀